To: vim_dev@googlegroups.com
Subject: Patch 7.3.474
Fcc: outbox
From: Bram Moolenaar <Bram@moolenaar.net>
Mime-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit
------------

Patch 7.3.474
Problem:    Perl build with gcc 4 fails.
Solution:   Remove XS() statements. (Yasuhiro Matsumoto)
Files:	    src/if_perl.xs


*** ../vim-7.3.473/src/if_perl.xs	2012-02-12 00:31:47.000000000 +0100
--- src/if_perl.xs	2012-03-16 19:33:23.000000000 +0100
***************
*** 913,936 ****
  win_T *win_find_nr(int n) { return curwin; }
  #endif
  
- XS(XS_VIM_Msg);
- XS(XS_VIM_SetOption);
- XS(XS_VIM_DoCommand);
- XS(XS_VIM_Eval);
- XS(XS_VIM_Buffers);
- XS(XS_VIM_Windows);
- XS(XS_VIWIN_DESTROY);
- XS(XS_VIWIN_Buffer);
- XS(XS_VIWIN_SetHeight);
- XS(XS_VIWIN_Cursor);
- XS(XS_VIBUF_DESTROY);
- XS(XS_VIBUF_Name);
- XS(XS_VIBUF_Number);
- XS(XS_VIBUF_Count);
- XS(XS_VIBUF_Get);
- XS(XS_VIBUF_Set);
- XS(XS_VIBUF_Delete);
- XS(XS_VIBUF_Append);
  XS(boot_VIM);
  
      static void
--- 913,918 ----
*** ../vim-7.3.473/src/version.c	2012-03-16 19:24:21.000000000 +0100
--- src/version.c	2012-03-16 19:28:03.000000000 +0100
***************
*** 716,717 ****
--- 716,719 ----
  {   /* Add new patch number below this line */
+ /**/
+     474,
  /**/

-- 
hundred-and-one symptoms of being an internet addict:
202. You're amazed to find out Spam is a food.

 /// Bram Moolenaar -- Bram@Moolenaar.net -- http://www.Moolenaar.net   \\\
///        sponsor Vim, vote for features -- http://www.Vim.org/sponsor/ \\\
\\\  an exciting new programming language -- http://www.Zimbu.org        ///
 \\\            help me help AIDS victims -- http://ICCF-Holland.org    ///
